packages/daimo-api/src/contract/coinIndexer.ts

```diff
@@ -132,6 +135,48 @@ export class CoinIndexer {
       txHash: transactionHash,
       logIndex,
       nonceMetadata,
+      opHash,
     };
   }
+
+  /* Attach fee amounts to transfer ops and filter out transfers involving
+   * paymaster.
+   */
+  private attachFeeAmounts(
```

Comment:
this would be a nice pure function to unit test

Reply:
good point, leaving as a todo for future since there's a good chance this code will go away soon anyway with free paymasters.